tests/test_datastructures.py

```diff
@@ -213,13 +213,13 @@ def test_queryparams():
     assert QueryParams(q) == q
 
 
-class TestUploadFile(UploadFile):
+class _TestUploadFile(UploadFile):
```

Comment:
I strongly dislike `__test__ = False` as it violates

https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers

> System-defined names, informally known as “dunder” names. These names are defined by the interpreter and its implementation (including the standard library). Current system names are discussed in the Special method names section and elsewhere. More will likely be defined in future versions of Python. **Any use of `__*__` names, in any context, that does not follow explicitly documented use, is subject to breakage without warning.**

there's other options here, like renaming to `ExampleUploadFile`, or nesting it under `test_upload_file()`

Reply:
Sure

Reply:
@JayH5 can you make a ` ```suggestion ` or push the change before merging? I don't want to bikeshed the name - just do the minimal fix

Reply:
```suggestion
class BigUploadFile(UploadFile):
```